tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-tox_pip_extensions_ext_venv_update=True
+# tox_pip_extensions_ext_venv_update=True
```

Comment:
Is there a reason we have this commented out?

Reply:
The following error occurs with `tox_pip_extensions_ext_venv_update=True`:
```
Error: version conflict: beautifulsoup4 4.8.2 (.tox/unittest-current/lib/python3.6/site-packages) <-> beautifulsoup4<4.6.1,>=4.5.1 (from wagtail==2.5.2->-r cfgov-refresh/requirements/wagtail.txt (line 1))
You are using pip version 18.1, however version 20.2b1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Reply:
Hmm. Okay. That's something we'll need to find a way around when we're doing future upgrades. `tox_pip_extensions_ext_venv_update` doesn't help us if we keep commenting it out.

Not something for this PR.

Reply:
I agree that we should find a way to address version conflict issues when upgrading to newer versions of Django and Wagtail. This sounds like a research task for a future sprint to determine what the best solution would be to resolve issues like to test against wagtail 2.9 we need a newer version of beautifulsoup4, but to use wagtail 2.5 an older version of beautifulsoup4 is required. This site may have some suggestions on best practices: https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires